frontend/src/pages/LogsPage/SearchForm/SearchForm.tsx

```diff
@@ -195,34 +232,39 @@ const Search: React.FC<{
 		>
 			<IconSolidSearch className={styles.searchIcon} />
 
-			<Combobox
-				autoSelect={autoSelect}
-				ref={inputRef}
-				state={state}
-				name="search"
-				placeholder="Search your logs..."
-				value={query}
-				onChange={(e) => {
-					const value = e.target.value
```

Comment:
I don't know where but I'm seeing a regression whereby clicking a key doesn't prepopulate until after a value is selected:

![Kapture 2023-03-07 at 16 03 07](https://user-images.githubusercontent.com/58678/223574109-e8814c40-465c-4aa5-ba4e-67fbd841a6ab.gif)

Reply:
Thanks for catching this! Fixed in [f90790c](https://github.com/highlight/highlight/pull/4524/commits/f90790c5e0273bfa568b0ea09de0142305536eb9)